src/features/user-dashboard/pages/UserListView.tsx

```diff
@@ -67,7 +67,7 @@ export const UserListView: FC = () => {
   );
 
   return (
-    <Container>
+    <FluidContainer>
```

Comment:
Fluid container is not super appropriate for the table views. Having a max width is still important as when pages get stretched out to 1920x1080 screens or above, the human eye doesn't really track across such a wide amount of horizontal space.

I think just regular `Container` here should be the default choice.